test/Truncate.js

```diff
@@ -497,6 +497,45 @@ describe('<Truncate />', () => {
             }
         });
 
+        describe('font', () => {
+            it('should use the native canvas font when no overrides are provided', () => {
+                const component = renderIntoDocument(
+                    <Truncate>
+                        abc
+                    </Truncate>
+                );
+
+                expect(component.canvasContext.font, 'to be', 'normal normal 10px sans-serif');
```

Comment:
This assertion looks a bit brittle, better check against `document.createElement('canvas').getContext('2d').font`